yoyodyne/sizing.py

```diff
@@ -0,0 +1,96 @@
+"""Finds the best batch size given constraints."""
+
+import argparse
+from typing import Tuple
+
+import pytorch_lightning as pl
+from pytorch_lightning.tuner import tuning
+
+from . import data, defaults, models
+
+
+def max_batch_size(
+    trainer: pl.Trainer,
+    model: models.BaseEncoderDecoder,
+    datamodule: data.DataModule,
+    *,
```

Comment:
I think we have been following the `*args` convention. Unless this is doing something else that I do not understand...

Reply:
This is unrelated to what you're mentioning. It is a convention in the language for requiring certain args to be passed as kwargs, and we use it elsewhere in the library.